RecoTracker/TkDetLayers/src/Phase2EndcapLayerDoubleDiskBuilder.cc

```diff
@@ -0,0 +1,21 @@
+#include "Phase2EndcapLayerDoubleDiskBuilder.h"
+#include "Phase2EndcapSubDiskBuilder.h"
+
+using namespace edm;
+using namespace std;
+
+Phase2EndcapLayerDoubleDisk* Phase2EndcapLayerDoubleDiskBuilder::build(const GeometricDet* aPhase2EndcapLayerDoubleDisk,
+                                                                       const TrackerGeometry* theGeomDetGeometry) {
+  LogTrace("TkDetLayers") << "Phase2EndcapLayerDoubleDiskBuilder::build";
+  vector<const GeometricDet*> theSubDisks = aPhase2EndcapLayerDoubleDisk->components();
```

Comment:
here in principle you are copying the vector. Is it large? Maybe better to do
```suggestion
  const auto& theSubDisks = aPhase2EndcapLayerDoubleDisk->components();
```